snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -35,16 +41,37 @@ def _run(self, command, hide_output: bool = False) -> None:
             instance_name=self.instance_name, command=command, hide_output=hide_output
         )
 
+    def _get_disk_image(self) -> str:
+        if self.project.info.base is not None and _get_platform() == "linux":
+            image = "file://{}".format(
+                get_cached_image_filepath(
+                    base=self.project.info.base, snap_arch=self.project.deb_arch
+                )
+            )
+        elif self.project.info.base == "core18":
+            image = "18.04"
+        elif self.project.info.base in ("core16", None):
+            image = "16.04"
+        else:
+            raise EnvironmentError(
```

Comment:
Should this be a snapcraft.EnvironmentError? Otherwise this string doesn't really get presented to them and it looks like it's a mistake.